<ui version="4.0" >
 <class>ChanMsgDialog</class>
 <widget class="QMainWindow" name="ChanMsgDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>528</height>
   </rect>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>Compose</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../images.qrc" >:/images/folder-draft.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QDockWidget" name="contactsdockWidget" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize" >
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="windowTitle" >
       <string>Contacts</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents" >
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <property name="horizontalSpacing" >
         <number>0</number>
        </property>
        <property name="verticalSpacing" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <widget class="QTreeWidget" name="msgSendList" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>4</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>160</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="sortingEnabled" >
           <bool>true</bool>
          </property>
          <column>
           <property name="text" >
            <string>Send To:</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label" >
            <property name="font" >
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:600; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Arial'; font-weight:400;">&lt;span style=" font-weight:600;">Subject:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="titleEdit" />
          </item>
         </layout>
        </item>
        <item row="1" column="0" >
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QComboBox" name="comboStyle" />
          </item>
          <item row="0" column="1" >
           <widget class="QFontComboBox" name="comboFont" />
          </item>
          <item row="0" column="2" >
           <widget class="QComboBox" name="comboSize" >
            <property name="toolTip" >
             <string>Font size</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QPushButton" name="boldbtn" >
            <property name="minimumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Bold</string>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="4" >
           <widget class="QPushButton" name="underlinebtn" >
            <property name="minimumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Underline</string>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="5" >
           <widget class="QPushButton" name="italicbtn" >
            <property name="minimumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Italic</string>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="6" >
           <widget class="QPushButton" name="colorbtn" >
            <property name="minimumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Color</string>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="7" >
           <widget class="QPushButton" name="textalignmentbtn" >
            <property name="minimumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Alignment</string>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="11" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>24</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="8" >
           <widget class="QPushButton" name="imagebtn" >
            <property name="minimumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Add a Image</string>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" >
         <widget class="QTextEdit" name="msgText" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="MinimumExpanding" hsizetype="Expanding" >
            <horstretch>4</horstretch>
            <verstretch>4</verstretch>
           </sizepolicy>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::TextEditorInteraction</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <property name="horizontalSpacing" >
         <number>6</number>
        </property>
        <property name="verticalSpacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QLabel" name="label_13" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text" >
           <string>Recommended Files</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QTreeWidget" name="msgFileList" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sortingEnabled" >
           <bool>true</bool>
          </property>
          <column>
           <property name="text" >
            <string>File Name</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Size</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Sources</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Hash</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="contextMenuPolicy" >
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionSend" />
   <addaction name="actionContactsView" />
   <addaction name="separator" />
   <addaction name="actionSaveas" />
  </widget>
  <action name="actionSend" >
   <property name="icon" >
    <iconset resource="../images.qrc" >:/images/send24.png</iconset>
   </property>
   <property name="text" >
    <string>Send</string>
   </property>
  </action>
  <action name="actionReply" >
   <property name="icon" >
    <iconset resource="../images.qrc" >:/images/replymail24.png</iconset>
   </property>
   <property name="text" >
    <string>Reply</string>
   </property>
  </action>
  <action name="actionContactsView" >
   <property name="text" >
    <string>Contacts</string>
   </property>
  </action>
  <action name="actionSaveas" >
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>msgSendList</tabstop>
  <tabstop>titleEdit</tabstop>
  <tabstop>msgText</tabstop>
  <tabstop>msgFileList</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc" />
 </resources>
 <connections/>
</ui>
